mysql-test/suite/innodb_fts/r/fulltext.result

```diff
@@ -784,3 +784,28 @@ f1	MATCH(f1) AGAINST ("test" IN BOOLEAN MODE)
 test	0.000000001885928302414186
 DROP TABLE t1;
 # End of 10.3 tests
+#
+#  MDEV-35183 ADD FULLTEXT INDEX unnecessarily DROPS FTS
+#		COMMON TABLES
+#
+CREATE TABLE t1 (
+ID INT UNSIGNED AUTO_INCREMENT NOT NULL PRIMARY KEY,
+title VARCHAR(200), book VARCHAR(200),
+FULLTEXT fidx(title)) ENGINE = InnoDB;
+INSERT INTO t1(title) VALUES('database');
+ALTER TABLE t1 DROP INDEX fidx;
+select space into @common_space from information_schema.innodb_sys_tables where name like "test/FTS_%_CONFIG";
+ALTER TABLE t1 ADD FULLTEXT fidx_1(book);
+select space=@common_space from information_schema.innodb_sys_tables where name like "test/FTS_%_CONFIG";
+space=@common_space
+0
```

Comment:
This should of course be `1`, and that is why the test is failing. Apparently you recorded this while the code change was absent.